Include/internal/pycore_code.h

```diff
@@ -472,6 +472,8 @@ typedef struct _PyShimCodeDef {
 extern PyCodeObject *
 _Py_MakeShimCode(const _PyShimCodeDef *code);
 
+extern uint32_t _Py_next_func_version;
```

Comment:
Is there a reason why this is called func_version and not code_version? I find it a bit confusing (note that in deepfreeze.py we now have ``_Py_next_func_version = {next_code_version}``).